services/staging/tests/conftest.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024 CS Group
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module used to configure pytests."""
+
+# Ignore not-at-top level import errors
+
+# pylint: disable=C0413
+# flake8: noqa: F402
+
+import os
+import os.path as osp
+from pathlib import Path
+
+# set pygeoapi env variables
+geoapi_cfg = Path(osp.realpath(osp.dirname(__file__))) / "test_config.yml"
```

Comment:
Is the "test_config.yml" file necessary ? It is the same as `../rs_server_staging/config/config.yml` maybe you could use this one instead ?

Reply:
config.yaml it's a configurable deployment file.

It will be moved to resources.